diff --git a/dist/index.mjs b/dist/index.mjs
index 8fc0df976b9bc71608350143988ac968482e8358..527677d0aa3676bc99dfee587fae60057acbe775 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3,10 +3,11 @@ import Debug from 'debug';
 import fs from 'fs';
 import chalk from 'chalk';
 
-const Prerenderer = require("@prerenderer/prerenderer");
-const PuppeteerRenderer = require("@prerenderer/renderer-puppeteer");
-const { minify } = require("html-minifier");
-const mkdirp = require("mkdirp");
+import Prerenderer from "@prerenderer/prerenderer";
+import PuppeteerRenderer from "@prerenderer/renderer-puppeteer";
+import { minify } from "html-minifier";
+import mkdirp from "mkdirp";
+
 const debug = Debug.debug("vite-plugin-prerender");
 const compilerFS = fs;
 function vitePrerender(options) {
@@ -70,7 +71,8 @@ const initOptions = (...args) => {
 };
 const emitRendered = (options) => {
   const PrerendererInstance = new Prerenderer(options);
-  PrerendererInstance.initialize().then(() => {
+  
+  return PrerendererInstance.initialize().then(() => {
     console.log(chalk.cyan(`[vite-plugin-prerender] Rendering routes [${chalk.green(`${options.routes.join(", ")}`)}] with puppeteer...`));
     return PrerendererInstance.renderRoutes(options.routes || []);
   }).then((renderedRoutes) => {
